uwmac0073.xml

```diff
@@ -829,113 +866,130 @@
           <did>
             <unittitle>Correspondence, 1970-1974</unittitle>
             <container type="box" label="Box">3</container>
-            <container type="folder" label="Folder">22</container>
+            <container type="folder" label="Folder">25</container>
           </did>
         </c02>
         <c02>
           <did>
             <unittitle>First and Second Year Work Program, 1970-1972</unittitle>
             <container type="box" label="Box">3</container>
-            <container type="folder" label="Folder">23</container>
+            <container type="folder" label="Folder">26</container>
           </did>
         </c02>
         <c02>
           <did>
             <unittitle>Reports, 1971-1972</unittitle>
             <container type="box" label="Box">3</container>
-            <container type="folder" label="Folder">24</container>
+            <container type="folder" label="Folder">27</container>
           </did>
         </c02>
       </c01>
       <c01>
         <did>
           <unittitle>Social Indicators II, Data Sheets--Originals, 1970-1972</unittitle>
           <container type="box" label="Box">3</container>
-          <container type="folder" label="Folder">25</container>
+          <container type="folder" label="Folder">28</container>
+        </did>
+      </c01>
+      <c01>
+        <did>
+          <unittitle><title render="italic">A Social Report for Milwaukee:
+          Trends and Indicators</title>, 1973</unittitle>
```

Comment:
Please use the spreadsheet to enter these when possible-- it ensures all the necessary encoding is there and it saves time besides. I now have to go through and encode these dates with <unitdate> myself.